packages/gui/src/components/plot/add/PlotAddNumberOfPlots.tsx

```diff
@@ -309,6 +309,17 @@ export default function PlotAddNumberOfPlots(props: Props) {
               />
             </FormControl>
           </Grid>
+          <Grid xs={12} item>
+            <FormControl variant="filled" fullWidth>
+              <TextField
+                name="plotNFTContractAddr"
+                type="text"
+                variant="filled"
+                placeholder="Plot NFT Plot Target Address, This will be used if a PlotNFT is not selected."
```

Comment:
This is too long for a placeholder string. I would change it to just
`Plot NFT Plot Target Address`, or add a tooltip with an explanation next to the TextField. I'm ignoring the punctuation/capitalization typos since we should be using a shorter string.

Reply:
Fixed